src/Google/Enumerators/BarcodeRenderEncoding.php

```diff
@@ -2,16 +2,23 @@
 
 namespace Chiiya\Passes\Google\Enumerators;
 
-final class BarcodeRenderEncoding
+use Chiiya\Passes\Common\LegacyValueEnumerator;
+
+final class BarcodeRenderEncoding implements LegacyValueEnumerator
 {
     /** @var string */
     public const RENDER_ENCODING_UNSPECIFIED = 'RENDER_ENCODING_UNSPECIFIED';
 
     /** @var string */
-    public const UTF_8 = 'UTF_8';
+    public const UTF_8 = 'utf8';
```

Comment:
For me it looks like an undocumented change, because the response from the fixture is grabbed from the API response and "anonymized".

I also sent a feedback to Google about the outdated docs.